Userland/Libraries/LibGUI/Window.cpp

```diff
@@ -454,7 +455,7 @@ void Window::handle_key_event(KeyEvent& event)
     }
 
     if (m_default_return_key_widget && event.key() == Key_Return)
-        if (!m_focused_widget || !is<Button>(m_focused_widget.ptr()))
+        if (!m_focused_widget || !is<GUI::Button>(m_focused_widget.ptr()))
```

Comment:
We're in namespace GUI here, this should work as is.

Reply:
> We're in namespace GUI here, this should work as is.

yes and no; it works when building for serenity but not in lagom yet. Personally I prefer being explicit so even if that weren't the case imo this is still an improvement.

Reply:
> Personally I prefer being explicit so even if that weren't the case imo this is still an improvement.

No, that's the exact opposite of the project style. Please adhere to it instead of trying to do your own thing, it there's a technical reason why it doesn't work yet (how's it any different in Lagom ?), that needs to be explained.

Reply:
> > Personally I prefer being explicit so even if that weren't the case imo this is still an improvement.
> 
> No, that's the exact opposite of the project style. 
wasn't aware that were the case when it comes to classes.

Currently I'm not sure why it's failing, I'll investigate.

Reply:
> Though, we pretty much use GUI::* everywhere else

I'm pretty sure that's a leftover from when things were called GWidget, GButton, etc. and a global search and replace took place. LibGUI is one of the oldest parts of the system after all :^)
Let's not add the prefix her, both to keep the diff smaller and work towards the new style of not using namespace prefix within their own namespace.

Reply:
> > Though, we pretty much use GUI::* everywhere else
> 
> I'm pretty sure that's a leftover from when things were called GWidget, GButton, etc. and a global search and replace took place. LibGUI is one of the oldest parts of the system after all :^) Let's not add the prefix here, both to keep the diff smaller and work towards the new style of not using namespace prefix within their own namespace.

Alright, I’ll revert that change! :^)